csharp/generate/templates/ApiClient.mustache

```diff
@@ -409,6 +405,8 @@ namespace {{packageName}}.Client
             var transformed = new ApiResponse<T>(response.StatusCode, new Multimap<string, string>({{#caseInsensitiveResponseHeaders}}StringComparer.OrdinalIgnoreCase{{/caseInsensitiveResponseHeaders}}), result, rawContent)
             {
                 ErrorText = response.ErrorMessage,
+                ResponseStatus = response.ResponseStatus,
+                InternalException = response.ErrorException,
```

Comment:
Is there anything on here that is more valuable than what is already on `ErrorMessage`?

Reply:
Yes, the stack trace of where the exception happened. Mind that the exception can happen anywhere in the `Deserialize` method, and we want to be able to pinpoint where it happened instead of returning a null

Reply:
Which in its current state will do just that - throw an error and subsequently return null in the RestSharp client if response is OK, but something fails in the processing